.github/workflows/pull-request.yaml

```diff
@@ -0,0 +1,14 @@
+name: Build PR
+on:
+  pull_request:
+
+jobs:
+  smoke-test:
```

Comment:
Should we define the jobs in one file to avoid copy-paste?

Reply:
Was thinking about the same thing. Decided to split them because those code bases will deviate in the future and combining those files requires ugly and hard to understand if-statements (think JSP) in the middle of the otherwise easy to understand action file. Let's just keep it simple.